wranglertools/import_data.py

```diff
@@ -1451,8 +1454,9 @@ def upload_file(creds, path):  # pragma: no cover
     # ~12-15s/GB from AWS Ireland - AWS Oregon
     print("Uploading file.")
```

Comment:
For this and other calls to print, I suggest importing `PRINT` from `dcicutils.misc_utils`. Not only does that have various handy ways to mock it for testing, but in the future I hope to add some unit tests in various libraries that look for stray `print` statements (which people often insert for debugging and forget to remove), so we've been following my made-up practice that seems to work decently of marking "intentional programmatic print" as `PRINT` and leaving `print` for "ephemeral debugging print". They should both do the same thing otherwise, take the same arguments, etc.

Reply:
Good to know. This goes beyond the scope of this PR, and I guess we want to do it systematically across the entire repo. Thanks for letting me know about this practice, we'll implement it in a future PR.